sw/device/silicon_creator/manuf/base/BUILD

```
@@ -261,10 +262,32 @@ cc_library(
         "//sw/device/silicon_creator/lib/cert:tpm_ek_template_library",
         "//sw/device/silicon_creator/lib/drivers:flash_ctrl",
         "//sw/device/silicon_creator/lib/drivers:hmac",
+        "//sw/device/silicon_creator/manuf/base:perso_tlv_data",
         "//sw/device/silicon_creator/manuf/lib:personalize",
     ],
 )
 
+cc_library(
+    name = "perso_tlv_data",
+    srcs = ["perso_tlv_data.c"],
+    deps = [
+        ":perso_tlv_headers",
+        "//sw/device/lib/base:status",
+        "//sw/device/lib/runtime:log",
+        "//sw/device/lib/testing/json:provisioning_data",
+        "//sw/device/silicon_creator/lib/cert",
+    ],
+)
+
+cc_library(
+    name = "perso_tlv_headers",
+    hdrs = ["perso_tlv_data.h"],
+    deps = [
+        "//sw/device/lib/base:status",
+        "//sw/device/lib/testing/json:provisioning_data",
+    ],
+)
+
```

Why there two libraries here? I think one would suffice:

```suggestion
cc_library(
    name = "perso_tlv_data",
    srcs = ["perso_tlv_data.c"],
    hdrs = ["perso_tlv_data.h"],
    deps = [
        "//sw/device/lib/base:status",
        "//sw/device/lib/runtime:log",
        "//sw/device/lib/testing/json:provisioning_data",
        "//sw/device/silicon_creator/lib/cert",
    ],
)
```

---

This is how I started, but inclusion of  "//sw/device/silicon_creator/lib/cert" upsets  bindgen because of some OTBN stuff this pulls in (at least this is how it looked), so I separated headers, but forgot to clean up `perso_tlv_data` dependencies, which I did now.